model/DeliveryFactory.php

```diff
@@ -212,7 +212,7 @@ protected function createDeliveryResource(
     ) {
 
         $properties[DeliveryAssemblyService::PROPERTY_DELIVERY_TIME]      = time();
-        $properties[DeliveryAssemblyService::PROPERTY_DELIVERY_RUNTIME]   = $serviceCall->toOntology();
+        $properties[DeliveryAssemblyService::PROPERTY_DELIVERY_RUNTIME]   = json_encode($serviceCall);
```

Comment:
it looks like a breaking change, 
what happens when we don't get a json object,

Reply:
This is why I added the fallback to the ContainerRuntime Service